kadas/app/milx/kadasmilxexportdialog.cpp

```diff
@@ -0,0 +1,100 @@
+/***************************************************************************
+  kadasmilxexportdialog.cpp
+  --------------------------------------
+  Date                 : September 2024
+  Copyright            : (C) 2024 by Damiano Lombardi
+  Email                : damiano@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#include <qgis/qgsproject.h>
+
+#include "kadasapplication.h"
+#include "kadasmainwindow.h"
+#include "kadas/gui/milx/kadasmilxlayer.h"
+#include <milx/kadasmilxexportdialog.h>
```

Comment:
```suggestion
#include "milx/kadasmilxexportdialog.h"
```

this is an internal header (not from an external lib)

Reply:
thanks too many copy paste, will search the others to